lib/fastlane/plugin/wpmreleasetoolkit/actions/ios/ios_register_ci_machine.rb

```diff
@@ -0,0 +1,71 @@
+module Fastlane
+  module Actions
+    class IosRegisterCiMachineAction < Action
+      def self.run(params)
+        return unless other_action.is_ci
+
+        require 'socket'
+        require 'open3'
+
+        hostname = Socket.gethostname
+                         .delete_suffix('.local')
+                         .delete_suffix('.shared')
+
+        device_uuid, = Open3.capture2("system_profiler SPHardwareDataType | awk '/Hardware UUID/ { print $NF }'")
```

Comment:
Would probably be safer to use `-json` to have `system_profiler` output JSON instead of plain text, and parse that output with `JSON.parse`, rather than relying on `awk` and textual parsing?

Reply:
TIL!